plugins/outputs/otlp/otlp.go

```diff
@@ -0,0 +1,379 @@
+package otlp
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"sort"
+	"time"
+
+	// TODO: this should be simplified
+	// Imports the OTLP client from the prometheus sidecar
```

Comment:
If it should be done, please do it! :-)

Reply:
Whoops! I left the comment in there after cleaning things up 😄, fixed.